apps/nextjs-website/src/app/solutions/[solutionSlug]/page.tsx

```diff
@@ -3,25 +3,16 @@ import {
   makeMetadataFromStrapi,
 } from '@/helpers/metadata.helpers';
 import { Metadata } from 'next';
-import { baseUrl, REVALIDATE_SHORT_INTERVAL } from '@/config';
+import { baseUrl } from '@/config';
 import { getSolution } from '@/lib/api';
 import SolutionTemplate from '@/components/templates/SolutionTemplate/SolutionTemplate';
-import { getSolutionsProps } from '@/lib/cmsApi';
 import { generateStructuredDataScripts } from '@/helpers/generateStructuredDataScripts.helpers';
 import { getItemFromPaths } from '@/helpers/structuredData.helpers';
 
 type Params = {
   solutionSlug: string;
 };
 
```

Comment:
Like we have discuss in meeting i will keep these pages static for now

```suggestion
// TODO: remove when solutions metadata will be managed in strapi
export const revalidate = REVALIDATE_SHORT_INTERVAL;
export async function generateStaticParams() {
  const solutions = await getSolutionsProps();
  return [...solutions].map(({ slug }) => ({
    solutionSlug: slug,
  }));
}

```